client/lib/plans/features-list.js

```diff
@@ -520,7 +520,7 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_UPLOAD_THEMES ]: {
 		getSlug: () => constants.FEATURE_UPLOAD_THEMES,
-		getTitle: () => i18n.translate( 'Install Themes' ),
+		getTitle: () => i18n.translate( 'Install themes' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Install Themes', { context: 'theme'} )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)